src/client/interfaces/IUserInfo.ts

```diff
@@ -0,0 +1,14 @@
+export interface IUserInfo {
```

Comment:
Let's call these something client side specific, otherwise developers working on the backend/server side could easily get confused as to which one to use.

Reply:
changed